README.md

```diff
@@ -134,3 +134,110 @@ the following ansible config:
+
+1. Building the tokens: The `paragon build-tokens` build tokens command compiles
+   the JSON style dictionaries into CSS files in the `paragon/css/` directory.
+   These are supposed to be committed to the repository.
+2. Building the SCSS: The build token CSS files are then compiled into a brand
+   theme in the dist folder using the `paragon build-scss` command. The contents
+   of the dist directory are what needs to uploaded to the CDN and will be
+   loaded by MFEs.
+
+Unlike SCSS variables, which are evaluated at build time and then applied to a
+theme, CSS variables are evaluated at run time. This means you can build a base
+Paragon theme that is using CSS variables and then you can swap out the
+stylesheet that defines all the variables to quickly change the theme at
+runtime.
+
+To use the custom theme, you no longer need to install the custom branding
+package, instead you provide the theme stylesheet URLs via the MFE config API.
+The built themes can be uploaded to a CDN and those URLs can be specified via
+the MFE config API in the following format:
+
+```json
+{
+  ...
+  "MFE_CONFIG": {
+    ...
+    "PARAGON_THEME_URLS": {
+      "core": {
+        "urls": {
+          "default": "https://cdn.jsdelivr.net/npm/@edx/paragon@$paragonVersion/dist/core.min.css",
+          "brandOverride": "https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@#brandVersion/dist/core.min.css"
+        }
+      },
+      "defaults": {
+        "light": "light"
+      },
+      "variants": {
+        "light": {
+          "urls": {
+            "default": "https://cdn.jsdelivr.net/npm/@edx/paragon@$paragonVersion/dist/light.min.css",
+            "brandOverride": "https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@$brandVersion/dist/light.min.css"
+          }
+        }
+      }
+    }
+  }
+}
+
+```
+
+The above are the vanilla Paragon stylesheets. The core stylesheets contain the
+bulk of the Paragon styling while the variants have the style variables that
+modify how the core theme applies. For both of those you have a `default` and a
+`brandOverride`. The default is the core theme whereas the `brandOverride` applies
+on top. This keeps the theme smaller since you don't need to include and update
+all the variables from the core theme.
+
+In the above example when configuring the theme for an instance, you would
+upload the `dist` folder contents to a CDN, and add the path to `core.min.css`
+and `light.min.css` to the appropriate places. You can also create other themes
+like a `dark` theme or a separate theme for each site.
+
+NOTE: CSS variables are technically called CSS custom properties but they are
+called variables here because that more closely corresponds with how they are
+being used here.
+
+## WIP: Automatically deploying a runtime theme to S3 via gitlab CI
+
+This repo has a GitLab CI configuration to automatically deploy to AWS S3 if
+configured.
+
+To use that you need to configure the environment variables that AWS CLI needs
```

Comment:
This is cool, but I'm curious why we're adding it. Is it needed by one of our clients? Does it satisfy a workflow that the tutor plugin isn't handling? Or is this to make it more generally useful?